src/api.jsx

```diff
@@ -4,8 +4,12 @@ const baseApi = axios.create({
     baseURL: "https://nc-news-c0co.onrender.com",
   });
 
-  export const getArticles = () => {
-    return baseApi.get("api/articles").then((response) => {
+  export const getArticles = (article_id) => {
+let endPointString = "api/articles"
+    if(article_id !== undefined){
```

Comment:
undefined is a falsy value, so this logic is saying if article id is not falsy, meaning, it's truthy then run the code. So you can just check that article_id exists. if(article_id) achieves the same thing